docs/migrating-from-asciidoc.adoc

```diff
@@ -0,0 +1,341 @@
+These are attributes that either no longer exist, work differently, or have better alternatives.
+
+[cols="<20,<20,<60"]
+|====
+|{ap} |{ad} |Notes
+
+
+|`big`, `small`, `underline`, `overline`, `line-through`, colors
+|_deprecated_
+|Character attributes to apply formatting directly. 
+Usually better to apply a role, then apply the formatting based on that role by using a stylesheet.
+
+// Checked
+|`indoc`
+.2+|_not implemented_
+.2+|Provides the name and directory of the current document. 
+(Distinct from `docname`, because `indoc` may be an included document, and `docname` is always the master document).
+No {ad} equivalent.
+
+// Checked
+|`indir`
+
+|`asciidoc`
+|`asciidoctor` 
+|{ap} sets `asciidoc` to show that it is the current processor. 
+{ad} sets `asciidoctor` instead.
+
+// Ref migration guide
+|`toc2`
+.2+|`toc`
+.2+|Combined in a single attribute, see <<toc>>.
+
+// Ref migration guide
+|`toc-placement`
+
+// Ref ap UG. I know it workls in ad tables; no ref to ad paragrphs so assume not implemented.
+|options="unbreakable"
+|
+|In {ad}, this only works for tables, not paragraphs.
+
+// No checked
+|options="pgwide"
+|_not implemented_
+|DocBook attribute to make tables full page width, whatever the current indent.
+No {ad} equivalent.
+
+// old table
+|`replacements2`
+|`post_replacements`
+|Renamed.
+
+// http://www.methods.co.nz/asciidoc/userguide.html#X39
+// not sure if a source block is exactly the same thing
+|`plaintext`
+|_not implemented_
+|Use a source block.
+
+// Not checked
+|specialwords
+|_not implemented_
+|In {ap}, applies special formatting to named text. 
+In {ad} use a role and stylesheet, or a user-defined attribute, or define a macro.
+
+// does this need an explanation?
+|presubs
+|-
+|Not required
+
+// old table says not, but...
+|halign, valign for table cells
+|Use new table notation `^.^\|` etc
+|See <<cell>>.
+
+|sgml
+|_not implemented_
+|No {ad} equivalent (SGML is so last century).
+
+// Actually I would like this. Did I see a sample extension for it?
+|showcomments
+|_not implemented_
+|Turn single line comments into DocBook "remark" elements.
+No {ad} equvalent.
+
+|tabsize
+|_not implemented_
+|An include option to replace tabs with spaces. 
+In {ad}, tabs are fixed at 4 spaces, see <<normalize-block-indentation>>.
+
+|====
+
+[[migrate-stylesheet]]
+=== Default HTML stylesheet
+The {ap} and Acidoctor stylesheets are completely compatible, because their formatting is based on the same role attributes.
+If you happen to prefer the {ap} stylesheet, you can use it by copying it from the {ap} "stylesheets" directory and telling {ad} to use it with `-a stylesheet=asciidoc.css`.
+
+// I may be doing something wrong. Can't believe no-one else has had this problem!
+
+IMPORTANT: The {ad} stylesheet (unlike the {ap} stylesheet) loads its font based icons from the web, therefore does not work well offline. To solve this, do not use inline icons <<inline-icons>>.
```

Comment:
I don't mind the warning about the use of web fonts, but the recommendation that follows isn't correct. There is a way to use local assets and that should be the focus of the recommendation.